File: backend-common/src/main/java/cc/jktu/apps/common/dao/entity/UserEntity.java
package cc.jktu.apps.common.dao.entity;

import cc.jktu.apps.common.dao.entity.BaseEntity;
import com.baomidou.mybatisplus.annotation.TableName;
import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

@Data
@ToString(callSuper = true)
@EqualsAndHashCode(callSuper = true)
@TableName("user")
public class UserEntity extends BaseEntity {

    private String username;
    @JsonIgnore
    private String password;

}

File: backend-common/src/main/kotlin/cc/jktu/apps/common/dao/entity/UserEntity.kt
package cc.jktu.apps.common.dao.entity

import com.baomidou.mybatisplus.annotation.TableName
import com.fasterxml.jackson.annotation.JsonIgnore

@TableName("user")
data class UserEntity(
    var username: String? = null,
    @JsonIgnore
    var password: String? = null
) : BaseEntity()
